### PR TITLE
User/vnagula/permissionchange

### PR DESCRIPTION
**Pull request by venkatnagula**
Documentation change:

A user can get/cancel their own jobs without additional permissions. a global admin/printer admin can get/cancel other user jobs

### CODE REVIEW

File: api-reference/beta/api/printjob-cancel.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Cancel a print job. Print jobs can be canceled only on behalf of a user, using d
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
 
-In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription and have a permission that grants [Get printer](printer-get.md) access.
+In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription. A user can cancel their own jobs or have a permission that grants [Get printer](printer-get.md) access to cancel other user jobs.
```
Comment (braedenp-msft) [inline api-reference/beta/api/printjob-cancel.md:21]: I'd recommend a slight change:

> In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription. Licensed users can cancel their own jobs, but the app must also have a permission that grants [Get printer](printer-get.md) access to cancel other users' jobs.

**Review by braedenp-msft — APPROVED** [review]

**Review by nilakhan — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/printjob-cancel.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ Cancel a print job. Print jobs can be canceled only on behalf of a user, using d
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
 
-In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription and have a permission that grants [Get printer](printer-get.md) access.
+In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription.
```
Comment (nilakhan) [inline api-reference/beta/api/printjob-cancel.md:21]: Cancel print job still requires Get Printer scope: 

https://microsoft.visualstudio.com/Universal%20Print/_git/UP.Service?path=/services/MPSGraph/src/Service/Helpers/SecurityScope/PFATTokenSecurityScopeHelper.cs&version=GBmaster&line=214&lineEnd=214&lineStartColumn=20&lineEndColumn=88&lineStyle=plain&_a=contents

File: api-reference/beta/api/printjob-cancel.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (venkatnagula) [reply api-reference/beta/api/printjob-cancel.md]: Can you point to the code where we are checking the permissions? I changed required scopes in Graph and IPPService code as well.  my commit https://microsoft.visualstudio.com/Universal%20Print/_git/UP.Service/commit/83b5eb76bc9b10bdf9f7fc8b4c9486a017156582

File: api-reference/beta/api/printjob-cancel.md (hunk 2 above)
Comment (venkatnagula) [reply api-reference/beta/api/printjob-cancel.md:21]: thanks for the link. I will fix the code to allow user to cancel their own jobs. I modified at certain places but not all it seems

File: api-reference/beta/api/printjob-get.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Retrieve the properties and relationships of a print job.
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
 
-In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription and have a permission that grants [Get printer](printer-get.md) or [Get printerShare](printershare-get.md) access depending upon whether printer or printerShare is being used.
+In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription. A user can get their own jobs or have a permission that grants [Get printer](printer-get.md) or [Get printerShare](printershare-get.md) access depending upon whether printer or printerShare is being used to get other user jobs.
```
[the remark this answers is not in the document]
Comment (venkatnagula) [reply api-reference/beta/api/printjob-get.md]: outdate comment

File: api-reference/beta/api/printjob-cancel.md (hunk 2 above)
Comment (venkatnagula) [reply api-reference/beta/api/printjob-cancel.md:21]: fixed the code

**Review by nilakhan — APPROVED** [review]

**Comment by JarbasHorst** [comment]
Hi @venkatnagula, I see that you have added the label "review requested" but I noticed that you haven't yet addressed the two open comments. Please, let me know if you have any questions regarding some of the feedback I provided.

**Comment by braedenp-msft** [comment]
> Hi @venkatnagula, I see that you have added the label "review requested" but I noticed that you haven't yet addressed the two open comments. Please, let me know if you have any questions regarding some of the feedback I provided.

@JarbasHorst Can you please check to make sure you published your review comments? I can't see them here.

If you already did, can you please link to them, because I can't seem to find them.

**Comment by JarbasHorst** [comment]
Hello @venkatnagula, @braedenp-msft,
thank you very much for reaching out. Indeed, I forgot to push the review comments 😳 Sorry for that!

**Comment by braedenp-msft** [comment]
> Hello @venkatnagula, @braedenp-msft, thank you very much for reaching out. Indeed, I forgot to publish the review comments 😳 Sorry for that! I've just published them. Please, take a look when you will have time.

No worries! Thanks for publishing them 😊

**Comment by venkatnagula** [comment]
> published

@JarbasHorst Can you take a look at the PR once again. I incorporated your feedback.

**Comment by venkatnagula** [comment]
> @Lauragra this PR is ready for final review. Please, merge PR [17227](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17227) into this branch to include my final edits.

Hi , Is there any pending things from my end to complete this PR ?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**